src/main/java/no/nav/fo/veilarbregistrering/oppfolging/adapter/OppfolgingClient.java

```diff
@@ -111,7 +113,8 @@ private Builder buildSystemAuthorizationRequestWithUrl(Client client, String url
             return client.target(url)
                     .request()
                     .header("SystemAuthorization", this.gammelSystemUserTokenProvider.getToken())
-                    .header(AUTHORIZATION, "Bearer " + this.gammelSystemUserTokenProvider.getToken());
+                    .header(AUTHORIZATION, "Bearer " + this.gammelSystemUserTokenProvider.getToken())
+                    .header("Nav-Call-Id", MDC.get(MDCConstants.MDC_CALL_ID));
```

Comment:
Tror ikke vi kaller der for nav-call-ID, bare callId

Reply:
Mulig jeg tar feil her, og blander header og logg-id

Reply:
Brukt litt om hverandre. `x_callid` er tydeligvis også en greie